### PR TITLE
Fix typos.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by leonardoalt — DISMISSED** [review]

**Review by christianparpart — DISMISSED** [review]
looks good to me, how did you find them. spell checker? :)

**Comment by chriseth** [comment]
Yes, I think the spell checker database was updated.

**Comment by chriseth** [comment]
Now it seems to be fixed.

File: docs/types.rst [hunk 1]
```diff
@@ -499,9 +499,9 @@ terminate the string literal. Newline only terminates the string literal if it i
 Hexadecimal Literals
 --------------------
 
-Hexademical Literals are prefixed with the keyword ``hex`` and are enclosed in double or single-quotes (``hex"001122FF"``). Their content must be a hexadecimal string and their value will be the binary representation of those values.
+Hexadecimal literals are prefixed with the keyword ``hex`` and are enclosed in double or single-quotes (``hex"001122FF"``). Their content must be a hexadecimal string and their value will be the binary representation of those values.
 
-Hexademical Literals behave like String Literals and have the same convertibility restrictions.
+Hexadecimal literals behave like string literals and have the same convertibility restrictions.
```
Comment (axic) [inline docs/types.rst:504]: I think `String Literals` here was supposed to be a link to the `String Literals` section.

**Comment by chriseth** [comment]
@ChrisChinchilla could you turn the terms axic mentioned above into links, please? Thanks!

**Pull request merged**